src/pages/terraform/cli/[[...page]].tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactElement } from 'react'
+import terraformData from 'data/terraform.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+
+const basePath = 'cli'
+const baseName = 'CLI'
+const product = terraformData as Product
+// TODO: productSlug should possibly actually be `terraform`,
+// TODO: but https://mktg-content-api.vercel.app/api/content/terraform/version-metadata?partial=true
+// TODO: does not return any version with "isLatest: true"
```

Comment:
Could you update this to be a block comment (`/** */`) not have duplicate `TODO`? I find it harder to read with each line starting with it. 🙏🏻

Reply:
ack yeah sorry, i know to do this but habits die hard! thanks for flagging ❤️